ftdc/custom_format.go

```diff
@@ -188,30 +201,35 @@ func flattenStruct(item reflect.Value) ([]float32, error) {
 // The function right now does not recursively walk data structures. We assume for now that the
```

Comment:
> does not recursively walk data structures

No longer accurate? Or am I misunderstanding?

Reply:
No longer accurate, updated. Thanks for spotting!